src/shared/bindings.ts

```diff
@@ -111,4 +114,20 @@ export class BindingUtils {
 
     return binding;
   }
+
+  /**
+   * Tries to fetch current bindings.json from GitHub
+   * If it can't be reached, it uses the local JSON file
+   */
+  private static async getBindingsJson() {
+    try {
+      const { data } = await axios({
+        method: "get",
+        url: configConstants.bindingsJsonUrl,
+      });
+      return data;
+    } catch (err) {
+      return defaultBindingsJson;
```

Comment:
Great that you kept in the default - i was thinking about the case where it goes away at some point or if there is some transient error.